BioCro/DARPA/biocro_regional_darpa_files/subset_weather.Rmd

```diff
@@ -0,0 +1,148 @@
+---
+title: "How to Subset Large Weather Data Files"
+author: "Author: Kristina Riemer"
+output: github_document
+urlcolor: blue
+---
+
+## Data file information
+
+This is for how to subset the NARR `all.nc` file. It is on Globus in the UA Field Scanner collection [here](https://app.globus.org/file-manager?origin_id=cbae3a96-d081-4951-bd1e-a8fda974cefa&origin_path=%2Fmet%2Fnarr%2Fthreehourly%2F). The file is ~650GB. This is not global, it is restricted to the northern hemisphere as below: 
+
+ * Latitudinal range: 10.125 - 83.875 
+ * Longitudinal range: -169.875 to -50.125
+
+The entire global file is the CRUNCEP `all.nc`, which is ~1.3TB and in the same Globus collection [here](https://app.globus.org/file-manager?origin_id=cbae3a96-d081-4951-bd1e-a8fda974cefa&origin_path=%2Fmet%2Fcruncep%2F). We will need more data space on the HPC to subset this because the upper limit is currently 1TB. 
```

Comment:
Maybe change "...subset this because the upper..." to "...subset this because its upper..."?